Assignment_college_document/index.php

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>College data</title>
+</head>
+<body>
+  <?php
+    include('College.php');
+    include('Data.php');
+    $obj = new College($college_data,$doc);
```

Comment:
The documents were supposed to be an array of objects. I am not seeing that here